io.sourceforge.zaz.yaml

```diff
@@ -0,0 +1,51 @@
+app-id: io.sourceforge.zaz
+runtime: org.freedesktop.Platform
+runtime-version: 22.08
+sdk: org.freedesktop.Sdk
+command: zaz
+rename-desktop-file: zaz.desktop
+rename-icon: zaz
+finish-args:
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --socket=fallback-x11
+  - --share=ipc
+  - --share=network
+  - --device=dri
+
+modules:
+  - shared-modules/glu/glu-9.json
+
+  - name: sdl12-compat
+    buildsystem: cmake-ninja
+    sources:
+      - type: archive
+        url: https://github.com/libsdl-org/sdl12-compat/archive/refs/tags/release-1.2.52.tar.gz
+        sha256: 5bd7942703575554670a8767ae030f7921a0ac3c5e2fd173a537b7c7a8599014
+
+  - shared-modules/SDL/SDL_image-1.2.12.json
+
+  - name: ftgl
```

Comment:
you never cleanup include and other development specific files.

Same with sdl12